### PR TITLE
Generating report on button

### PR DESCRIPTION
**Pull request by Kaszana102**
at the moment it doesn't generate report, because it has not  been merged yet. However it works with the file you set the path to.  Having set the path, a new tab will open with the **pdf** file content.

### CODE REVIEW

File: src/manifest.py [hunk 1]
```diff
@@ -4,13 +4,14 @@
 COMPANY_NIP = "123-456-78-90"
 COMPANY_BANK_ACCOUNT = "12 3456 7890 0000 0000 1234 5678"
 
+# name should be in female form
```
Comment (Lenetis) [inline src/manifest.py]: This comment is not necessary and will probably be confusing outside the context of this commit

**Review by Lenetis — APPROVED** [review]
Approved, but I think we should wait with merging until generating reports is merged

File: src/manifest.py (hunk 1 above)
Comment (Kaszana102) [reply src/manifest.py]: it's right above the subscriptions, the subscriptions name ought to be in female form

Comment (Lenetis) [reply src/manifest.py]: Yes, and it is in female form now. What is the use of this comment after the change?
There isn't a "#name should be in male form" comment above packets, and this comment is also unnecessary.

Comment (Kaszana102) [reply src/manifest.py]: FIXED COMMENT

Comment (Lenetis) [reply src/manifest.py]: I wanted the comment to be removed, but I'm ok with this change too

File: functional/admin_panel_test.py [hunk 2]
```diff
@@ -36,13 +36,15 @@ def test_send_invoice(self):
...
 
-        self.driver.find_element(by=By.ID, value="generate-report-label").click()
-
-        assert self.driver.find_element(by=By.ID, value="notification").text == "Report generated"
-
+        self.driver.find_element(by=By.ID, value="generate-report-button").click()
+        handles = self.driver.window_handles
+        self.driver.switch_to.window(handles[1])
+        assert self.driver.title == 'Paint Drying/Admin Panel/Report'
+    '''
```
Comment (BognaLew) [inline functional/admin_panel_test.py]: Why this test is in comment?

**Review by BognaLew — APPROVED** [review]

**Pull request merged**